mls-rs/src/group_joiner.rs

```diff
@@ -337,39 +372,54 @@ mod tests {
             .await
             .unwrap();
 
-        let commit = bob
+        let mut bob_group = bob
             .create_group(Default::default(), Default::default())
             .await
-            .unwrap()
+            .unwrap();
+
+        let commit = bob_group
             .commit_builder()
             .add_member(kp_alice.key_package_message)
             .unwrap()
-            .add_external_psk(psk_id.clone())
+            .add_external_psk(psk_id.clone(), psk_secret.clone())
             .unwrap()
-            .add_resumption_psk_for_group(1, group_alice.group_id().to_vec())
+            .add_resumption_psk_for_group(
+                1,
+                group_alice.group_id().to_vec(),
+                resumption_psk.clone(),
+            )
             .unwrap()
             .build()
             .await
             .unwrap();
 
         let joiner = alice
             .group_joiner(&commit.welcome_messages[0], kp_alice.key_package_data)
-            .await
-            .unwrap();
+            .unwrap()
+            .await;
```

Comment:
how is the sequence `unwrap().await` possible here?

Reply:
fixed